lib/urlwatch/config.py

```diff
@@ -93,6 +93,7 @@ def parse_args(self, cmdline_args):
         group.add_argument('--delete', metavar='JOB', help='delete job by location or index')
         group.add_argument('--enable', metavar='JOB', help='enable job by location or index')
         group.add_argument('--disable', metavar='JOB', help='disable job by location or index')
+        group.add_argument('--run-new-jobs', action='store_true', help='run new jobs or jobs without history')
```

Comment:
As you pointed out, it's "new jobs or jobs without history", so maybe it can be something else than `--run-new-jobs`? What about e.g. `--bootstrap-jobs` or `--initialize-jobs` or `--request-first-version` or something (I don't realy like any of them particularly much, but maybe it helps guide you in the right direction).

Reply:
Yea, i wanted to make it as obviously as it can get.  `--initialize-jobs` is from the three the most fitting in my opinion. 
Or maybe `--prepare-jobs` since its easier to type.